.openpublishing.publish.config.json

```diff
@@ -7,6 +7,11 @@
       "build_output_subfolder": "ef",
       "locale": "en-us",
       "open_to_public_contributors": true,
+      "xref_query_tags": [
```

Comment:
@ajcvickers  this change allows the repo to use xref tags like I added to the new doc
`<xref:Microsoft.EntityFrameworkCore.DbContext>`